### PR TITLE
Working

### PR DESCRIPTION
**Pull request by Benjaminpjacobs**
Hey Regis,

I'm not sure if I'm doing this PR part correctly but I did create a second "working" branch where all of my current commits are. After you review it, I'll merge it with the master. Lets see if I did this right...

Thanks!

Ben

### CODE REVIEW

**Comment by Benjaminpjacobs** [comment]
@selfup - I believe this actually tags you....

File: parking_lot.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require_relative "./car.rb"
+
+class ParkingLot
+  def initialize (filename)
+    @cars_hash = {}
+    cars = File.open(filename, "r")
+    cars_array = cars.read.split("\n\n\n").map{|words| words.split}.map{|car| @cars_hash[car[0]] = car[1]}
```
Comment (selfup) [inline parking_lot.rb]: Cars array should be an instance variable pointing to a method 🎉 

This is great but try to keep lines under 80 characters. You can chain methods in ruby like this:

```ruby
cars_array = cars.read.split("\n\n\n")
  .map { |words| words.split }
  .map { |car| @cars_hash[car[0]] = car[1] }
```

Comment (selfup) [reply parking_lot.rb]: Also it would be better to store the `Car.new(key, value)` in the hash here 😄

**Comment by selfup** [comment]
For a project with no real spec, this is a **great* first start!

File: parking_lot.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    puts @cars_hash
+  end 
+
+  def list_cars
+    @cars_hash.each do |key, value|
+      key = Car.new(key, value)
```
Comment (selfup) [inline parking_lot.rb]: Once you extract `Car.new` to hash builder, now you just have to call `.what_car` here and can make it a one liner!

**Pull request closed without merging**